manifests/charts/gateways/istio-ingress/templates/meshexpansion.yaml

```diff
@@ -37,6 +44,13 @@ spec:
         host: istiod.{{ .Values.global.istioNamespace }}.svc.{{ .Values.global.proxy.clusterDomain }}
         port:
           number: 15012
+  - match:
+    - port: 15017
+    route:
+    - destination:
+        host: istiod.{{ .Release.Namespace }}.svc.{{ .Values.global.proxy.clusterDomain }}
```

Comment:
nit: do we need proxy.clusterDomain ? Nate is trying to move it to mesh config as well.

Reply:
It was come up as @morvencao 's comment to not hard code things.  If it is configured in mesh config, it should be fine.